### DISCUSSION TITLE
Why is the + important?

### DISCUSSION THREAD

**Discussion by maxfreu** [tbnobody/OpenDTU#357] [Q&A] [answered]
Hi!
By accident I ordered a nrf24l01 without + (the plus in the description was somewhere else). So: Why is the + important? Should I return the chip I bought?
Kind regards
Max

**Comment by tbnobody** [comment, marked as answer]
Only the plus variant supports the required transmission speed of 250k.

**Comment by maxfreu** [comment]
Ok thanks, so a return it is...

**Reply by stefan123t** [reply]
Did you check the Rf24.printPrettyDetails() output for the model name and RF Data Rate or have you sent it back already ?
